docker-compose.yaml

```diff
@@ -1,34 +1,151 @@
-version: "3.3"
+version: "2"
```

Comment:
Is there a reason you switched this to use version 2?

Reply:
I have a machine that I'm testing this on, but it's using v2